kodilla-testing/build.gradle

```diff
@@ -10,5 +10,8 @@ repositories {
 }
 
 dependencies {
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    //External dependencies
+    //compile('org.springframework.boot:spring-boot-starter-web')
+    //testCompile group: 'junit', name: 'junit', version: '4.12'
+    testCompile ('junit:junit:4.12')
```

Comment:
`spring-boot` - jak nie używany, to po prostu powinien być usunięty.
`testCompile group: 'junit', name: 'junit', version: '4.12'` to dokładnie to samo co ` testCompile ('junit:junit:4.12')` (inna forma zapisu) - więc tę zakomentowany wariant też można usunąć.

Reply:
ok